shell/bashrc

```diff
@@ -319,7 +324,6 @@ function phpbrew ()
             unset PHPBREW_PATH
             eval `$BIN env`
             __phpbrew_set_path
-            echo "phpbrew is turned off."
```

Comment:
another option is we can add a new command for iteratal operations with shell commands support, and keep `each` syntax. but I quite doubt if there is someone use `each` with phpbrew commands only, since the extensions and apps are bundled with specific php version, i.e. `phpbrew each ext install xdebug` would definately fail on either 5.3 or 7.0.

any thoughts? @c9s